internal/pkg/global/config.go

```diff
@@ -146,6 +149,7 @@ type PlatformConfig struct {
 	Addr               string        `yaml:"addr"`
 	URI                string        `yaml:"uri"`
 	RetryCount         int           `yaml:"retry_count"`
+	IsEnabled          *bool         `yaml:"enabled"`
```

Comment:
Please rename to `Enabled` for consistency.

Reply:
Addressed in 6d05b87